src/main/kotlin/no/nav/amt/aktivitetskort/service/AktivitetskortService.kt

```diff
@@ -0,0 +1,101 @@
+package no.nav.amt.aktivitetskort.service
+
+import no.nav.amt.aktivitetskort.domain.Aktivitetskort
+import no.nav.amt.aktivitetskort.domain.Arrangor
+import no.nav.amt.aktivitetskort.domain.Deltaker
+import no.nav.amt.aktivitetskort.domain.Deltakerliste
+import no.nav.amt.aktivitetskort.domain.Detalj
+import no.nav.amt.aktivitetskort.domain.EndretAv
+import no.nav.amt.aktivitetskort.domain.IdentType
+import no.nav.amt.aktivitetskort.domain.Melding
+import no.nav.amt.aktivitetskort.repositories.ArrangorRepository
+import no.nav.amt.aktivitetskort.repositories.DeltakerRepository
+import no.nav.amt.aktivitetskort.repositories.DeltakerlisteRepository
+import no.nav.amt.aktivitetskort.repositories.MeldingRepository
+import no.nav.amt.aktivitetskort.service.StatusMapping.deltakerStatusTilAktivetStatus
+import no.nav.amt.aktivitetskort.service.StatusMapping.deltakerStatusTilEtikett
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.util.UUID
+import kotlin.RuntimeException
+
+@Service
+class AktivitetskortService(
+	private val meldingRepository: MeldingRepository,
+	private val arrangorRepository: ArrangorRepository,
+	private val deltakerlisteRepository: DeltakerlisteRepository,
+	private val deltakerRepository: DeltakerRepository,
+) {
+
+	fun lagAktivitetskort(deltaker: Deltaker): List<Aktivitetskort> {
+		val eksisterendeMelding = meldingRepository.getByDeltakerId(deltaker.id)
+
+		val melding = if (eksisterendeMelding == null) {
+			opprettMelding(deltaker, UUID.randomUUID())
+		} else {
+			opprettMelding(deltaker, eksisterendeMelding.aktivitetskort.id)
+		}
+
+		return listOf(melding.aktivitetskort)
```

Comment:
Hvorfor retrurnerer denne en liste?

Reply:
Fordi jeg vil at alle `lagAktivitetskort` skal være lik, slipper å ha flere funskjoner for å jobbe videre med aktivitetskortet, men jeg skjønner hvorfor en vil stusse over det. Kan se på å forenkle det.